### ISSUE TITLE
Search in CVEs for My Vendors / Products

### ISSUE THREAD

**Issue by gitmopp** [opencve/opencve#152]
Currently there is no direct interface to search for products or vendors.
It works when you find the product in the list and click on the link, e.g., https://www.opencve.io/cve?vendor=fedoraproject&product=fedora
Of course it is possible to change it in the URI, but there is no search field for it or the list to My Vendors / My Products which would make it much more user friendly.

It would be nice if there would be a drop down list to search for CVEs in my subsciption list.

**Comment by M0dEx** [comment]
Hi there!

You can search for vendors using the general search field. If you search for "Fedora" all CVE which have "Fedora" in their summary or in the CPEs will be returned. If you then want to see only CVEs for the specific product "Fedora", just click on the hyperlink "Fedora" on any of the returned CVEs and that should filter the search results just for that product.

**Comment by gitmopp** [comment]
> Hi there!
> 
> You can search for vendors using the general search field. If you search for "Fedora" all CVE which have "Fedora" in their summary or in the CPEs will be returned. If you then want to see only CVEs for the specific product "Fedora", just click on the hyperlink "Fedora" on any of the returned CVEs and that should filter the search results just for that product.

Yes, you are completly right. But it would be nice to have a search field for it or the drop downlist with my subscriptions to get an quick overview.
I already looked at the source code. I think it is easy to implement if you understand how the whole thing works and this is currently my problem.

**Comment by M0dEx** [comment]
> > Hi there!
> > 
> > You can search for vendors using the general search field. If you search for "Fedora" all CVE which have "Fedora" in their summary or in the CPEs will be returned. If you then want to see only CVEs for the specific product "Fedora", just click on the hyperlink "Fedora" on any of the returned CVEs and that should filter the search results just for that product.
> 
> Yes, you are completly right. But it would be nice to have a search field for it or the drop downlist with my subscriptions to get an quick overview.
> I already looked at the source code. I think it is easy to implement if you understand how the whole thing works and this is currently my problem.

I think I could take a look at this, if @ncrocfer  agrees.

**Comment by ncrocfer** [comment]
Yes with pleasure @M0dEx  !

**Comment by dagobertdebug** [comment]
+1 for that feature

**Comment by ncrocfer** [comment]
Closing this issue as it has been inactive for a long time.
